### PR TITLE
MRG+1: Fix information about info

### LINKED ISSUE #2287: Document info data structure
For being such an important data structure, there sure isn't any documentation about the `Info` class. What fields might it contain? What do they mean? I know the answer often is 'it depends', but the standard fields should be documented.

### PR DESCRIPTION
**Pull request by Eric89GXL**
Just filling out some last bits of information about `Info`.

Closes #2287.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Also flattened the namespaces a bit. And fixed end-of-line characters in fieldtrip_client.py.

File: mne/realtime/fieldtrip_client.py [hunk 1]
```diff
@@ -1,351 +1,351 @@
-# Author: Mainak Jas
```
Comment (agramfort) [inline mne/realtime/fieldtrip_client.py]: what did you change here?

**Comment by agramfort** [comment]
that's it for me here

File: mne/realtime/fieldtrip_client.py (hunk 1 above)
Comment (Eric89GXL) [inline mne/realtime/fieldtrip_client.py]: The line endings were incorrect/mixed. Now they should be consistent with the rest of the repo.

Comment (agramfort) [inline mne/realtime/fieldtrip_client.py]: ok

**Comment by agramfort** [comment]
LGTM

any second reviewer?

**Comment by dengemann** [comment]
@Eric89GXL what's going on here? Info is moved to the mne namespace? No updates of tutorials? No deprecation? I seem to overlook something.

**Comment by Eric89GXL** [comment]
No it's in the `io` namespace for convenience, is still in `io.meas_info` namespace so no need for deprecation, and docs have been updated

**Comment by dengemann** [comment]
Ah ok, what about the tutorials that @wmvanvliet did over the sprint?

**Comment by Eric89GXL** [comment]
Which are those?

**Comment by Eric89GXL** [comment]
(I wasn't there / don't remember the PR)

**Comment by dengemann** [comment]
Checked it it's fine. Thanks Eric!

File: mne/realtime/fieldtrip_client.py (hunk 1 above)
Comment (jasmainak) [inline mne/realtime/fieldtrip_client.py]: is it possible to have a test for the line endings to test the entire repo?

Comment (Eric89GXL) [inline mne/realtime/fieldtrip_client.py]: Yeah I can add one in another PR

Comment (Eric89GXL) [inline mne/realtime/fieldtrip_client.py]: (After this is merged)

**Comment by wmvanvliet** [comment]
yay, no more mysteries regarding the Info object. Thanks Eric!

**Pull request merged**